modules/standard/Path.chpl

```diff
@@ -573,13 +684,21 @@ proc normPath(name: string): string {
    :return: A canonical version of the argument.
    :rtype: `string`
 */
-proc realPath(name: string): string throws {
+inline proc realPath(name: string): string throws {
```

Comment:
This one must always return `bytes`, because you could have a symbolic link that refers to a non-UTF-8-encodable file name. Anything can return a file name read from the system should return bytes.